lib/tasks/gather.js

```diff
@@ -177,10 +177,8 @@ class TaskGather extends SttTask {
     this._startVad();
 
     const startDtmfListener = () => {
-      if (this._dtmfListenerStarted) return;
```

Comment:
if we are confident we are not going to call this twice, then I would still like to see an assert here

Reply:
I see
fixed